src/server/schema/resolvers/queries/getLife.ts

```diff
@@ -0,0 +1,19 @@
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers } from '../definitions';
+
+const query: GraphQLQueryResolvers['getLife'] = async (root, { id }, context, info) => {
+    const { collections } = await getDatabaseContext();
+
+    // Fetch the documents as an array
+    const life = await collections.lives.findOne({ _id: id });
+    // Transform each MongoDB document to match the GraphQLLife type
+    const transformedLives = {
+        ...life,
+        id: life._id,
+        fullName: `${life.firstName} ${life.lastName}`, // Dynamically create fullName
```

Comment:
We will not do it here, rather, we will do it in `Life` resolver

Reply:
same goes for the `._id`, you can refer to `User` resolver regarding implementation

Reply:
Fix on newly commit